src/components/repositories-list/repositories-list.js

```diff
@@ -4,27 +4,27 @@ import styled from 'styled-components'
 import { MdStar } from 'react-icons/md'
 import EmptyList from '../empty-list'
 
-const RepositoriesList = ({ className, repos }) => {
+const RepositoriesList = ({ className, repositories }) => {
   return (
     <React.Fragment>
       <h2>Lista de repositórios</h2>
       <ul className={className}>
-        {repos.length > 0 ? repos.map(repo => <li key={repo.name}>
+        {repositories.length > 0 ? repositories.map(repositorie => <li key={repositorie.name}>
```

Comment:
you should change variable name `repositorie` to another... what are u think anout this? :)